test/e2e/common/node/pod_resize.go

```diff
@@ -1196,17 +1196,18 @@ func doPodResizeTests() {
 			framework.ExpectNoError(e2epod.VerifyPodContainersCgroupValues(ctx, f, newPod, tc.containers))
 
 			patchAndVerify := func(patchString string, expectedContainers []e2epod.ResizableContainerInfo, opStr string) {
+				expected := e2epod.UpdateExpectedContainerRestarts(ctx, podClient, newPod.Name, expectedContainers)
 				ginkgo.By(fmt.Sprintf("patching pod for %s", opStr))
 				patchedPod, pErr = f.ClientSet.CoreV1().Pods(newPod.Namespace).Patch(ctx, newPod.Name,
```

Comment:
You could just use `patchedPod` here to read the current restart count, rather than issuing a separate get request. The patched pod is returned by the apiserver when the patch is successfully written to storage, so this will be before the Kubelet has gotten the change.

Reply:
thaaaank you, done!